### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by s-laugh**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/general.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 name: General
-about: This a general pull request template to be used for any reason
+about: This a general pull request template to be used as a default template
```
Comment (PhilippeTurcotteGov) [inline .github/PULL_REQUEST_TEMPLATE/general.md]:
```suggestion
about: This is a general pull request template to be used as a default template
```

File: .github/ISSUE_TEMPLATE/potential-devcop-topic.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+### Goals / Recommendation
+
+<!-- What is is we would be achieving by presenting this topic. Does it come with any recommendations? -->
+
+### Community Expectations
+
+<!-- What are the exceptions you have of the community. Do they need some prior knowledge? -->
```
Comment (PhilippeTurcotteGov) [inline .github/ISSUE_TEMPLATE/potential-devcop-topic.md]:
```suggestion
<!-- What expectations do you have of the community. Do they need some prior knowledge? -->
```

**Comment by s-laugh** [comment]
@PhilippeTurcotteGov I had to re-sign all the commits, which broke the review. Can you please re-review. No changes.

**Comment by PhilippeTurcotteGov** [comment]
@s-laugh done!

**Pull request merged**